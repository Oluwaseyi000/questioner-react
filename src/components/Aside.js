import React from 'react';

const Aside = () => {
  return (
    <aside className="aside">
      <div className="aside-header">Upcoming Meetups</div>
      <div className="aside-body">
        <div className="aside-content upcoming-meetup">
          <p>
            {' '}
            Today: <span> 0</span>
          </p>
          <p>
            {' '}
            This Week:<span> 23</span>
          </p>
          <p>
            {' '}
            This Month:<span> 123</span>
          </p>
        </div>
        <div className="aside-header"> Your Stats</div>
        <div className="aside-body">
          <div className="aside-content">
            <p className="stats-list">
              Question Posted: <span>10</span>{' '}
            </p>
            <p className="stats-list">
              {' '}
              Question Commented On:<span> 23</span>{' '}
            </p>
            <p className="stats-list">
              {' '}
              Total Upvote:P <span>123 </span>{' '}
            </p>
            <p className="stats-list">
              Total Downvote: <span>34 </span>{' '}
            </p>
          </div>
        </div>
      </div>
      <div className="aside-header">ADVERTS</div>
      <div className="aside-body">
        <img src="https://i.imgur.com/XaKeYJE.png" alt="advert" />
      </div>
    </aside>
  );
};

export default Aside;
