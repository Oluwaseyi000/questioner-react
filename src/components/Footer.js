import React from 'react';

const Footer = () => {
  return (
    <footer className="footer">
      copyright @ Adebajo Oluwaseyi | Andela 2018
    </footer>
  );
};

export default Footer;
