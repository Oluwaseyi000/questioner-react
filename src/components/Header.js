import React from 'react';
import { Link } from 'react-router-dom';

const Header = () => {
  return (
    <header className="header">
      <h1 className="header-logo">
        <Link to="/">
          <span className="q">Q</span>
          <span className="uestioner">UESTIONER</span>{' '}
        </Link>{' '}
      </h1>
      <div className="profile">
        <Link className="thelogin" to="login">
          LOG IN
        </Link>
        <Link className="thelogin" to="signup">
          SIGN UP
        </Link>
      </div>
    </header>
  );
};

export default Header;
