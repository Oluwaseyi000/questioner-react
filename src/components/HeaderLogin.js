import React from 'react';

const HeaderLogin = () => {
  return (
    <header className="header">
      <h1 className="header-logo">
        <a href="../index.html">
          <span className="q">Q</span>
          <span className="uestioner">UESTIONER</span>
        </a>{' '}
      </h1>

      <div className="profile">
        <a id="adminStatus" className="thelogin" href="../admin/meetups.html">
          ADMIN{' '}
        </a>
        <img
          id="dropdown"
          src="https://cdn.pixabay.com/photo/2016/08/08/09/17/avatar-1577909_960_720.png"
          alt=""
        />
        <br /> <span className="time" id="currentTime" />
      </div>
    </header>
  );
};

export default HeaderLogin;
