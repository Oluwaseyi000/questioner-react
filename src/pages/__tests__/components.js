import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { MemoryRouter as Router } from 'react-router-dom';
import Aside from '../../components/Aside';
import Footer from '../../components/Footer';
import Header from '../../components/Header';
import HeaderLogin from '../../components/HeaderLogin';

afterEach(cleanup);

describe('Should render components', () => {
  test('<Aside /> component', () => {
    const aside = render(<Aside />);
    expect(aside).toBeTruthy();
    expect(aside).toMatchSnapshot();
  });
  test('<Footer /> component', () => {
    const footer = render(<Footer />);
    expect(footer).toBeTruthy();
    expect(footer).toMatchSnapshot();
  });
  test('<Header /> component', () => {
    const header = render(
      <Router>
        <Header />
      </Router>,
    );
    expect(header).toBeTruthy();
    expect(header).toMatchSnapshot();
  });
  test('<HeaderLogin /> component', () => {
    const headerLogin = render(
      <Router>
        <HeaderLogin />
      </Router>,
    );
    expect(headerLogin).toBeTruthy();
    expect(headerLogin).toMatchSnapshot();
  });
});
