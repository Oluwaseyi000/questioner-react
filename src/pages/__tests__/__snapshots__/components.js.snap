// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render components <Aside /> component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <aside
        class="aside"
      >
        <div
          class="aside-header"
        >
          Upcoming Meetups
        </div>
        <div
          class="aside-body"
        >
          <div
            class="aside-content upcoming-meetup"
          >
            <p>
               
              Today: 
              <span>
                 0
              </span>
            </p>
            <p>
               
              This Week:
              <span>
                 23
              </span>
            </p>
            <p>
               
              This Month:
              <span>
                 123
              </span>
            </p>
          </div>
          <div
            class="aside-header"
          >
             Your Stats
          </div>
          <div
            class="aside-body"
          >
            <div
              class="aside-content"
            >
              <p
                class="stats-list"
              >
                Question Posted: 
                <span>
                  10
                </span>
                 
              </p>
              <p
                class="stats-list"
              >
                 
                Question Commented On:
                <span>
                   23
                </span>
                 
              </p>
              <p
                class="stats-list"
              >
                 
                Total Upvote:P 
                <span>
                  123 
                </span>
                 
              </p>
              <p
                class="stats-list"
              >
                Total Downvote: 
                <span>
                  34 
                </span>
                 
              </p>
            </div>
          </div>
        </div>
        <div
          class="aside-header"
        >
          ADVERTS
        </div>
        <div
          class="aside-body"
        >
          <img
            alt="advert"
            src="https://i.imgur.com/XaKeYJE.png"
          />
        </div>
      </aside>
    </div>
  </body>,
  "container": <div>
    <aside
      class="aside"
    >
      <div
        class="aside-header"
      >
        Upcoming Meetups
      </div>
      <div
        class="aside-body"
      >
        <div
          class="aside-content upcoming-meetup"
        >
          <p>
             
            Today: 
            <span>
               0
            </span>
          </p>
          <p>
             
            This Week:
            <span>
               23
            </span>
          </p>
          <p>
             
            This Month:
            <span>
               123
            </span>
          </p>
        </div>
        <div
          class="aside-header"
        >
           Your Stats
        </div>
        <div
          class="aside-body"
        >
          <div
            class="aside-content"
          >
            <p
              class="stats-list"
            >
              Question Posted: 
              <span>
                10
              </span>
               
            </p>
            <p
              class="stats-list"
            >
               
              Question Commented On:
              <span>
                 23
              </span>
               
            </p>
            <p
              class="stats-list"
            >
               
              Total Upvote:P 
              <span>
                123 
              </span>
               
            </p>
            <p
              class="stats-list"
            >
              Total Downvote: 
              <span>
                34 
              </span>
               
            </p>
          </div>
        </div>
      </div>
      <div
        class="aside-header"
      >
        ADVERTS
      </div>
      <div
        class="aside-body"
      >
        <img
          alt="advert"
          src="https://i.imgur.com/XaKeYJE.png"
        />
      </div>
    </aside>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Should render components <Footer /> component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <footer
        class="footer"
      >
        copyright @ Adebajo Oluwaseyi | Andela 2018
      </footer>
    </div>
  </body>,
  "container": <div>
    <footer
      class="footer"
    >
      copyright @ Adebajo Oluwaseyi | Andela 2018
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Should render components <Header /> component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header
        class="header"
      >
        <h1
          class="header-logo"
        >
          <a
            href="/"
          >
            <span
              class="q"
            >
              Q
            </span>
            <span
              class="uestioner"
            >
              UESTIONER
            </span>
             
          </a>
           
        </h1>
        <div
          class="profile"
        >
          <a
            class="thelogin"
            href="/login"
          >
            LOG IN
          </a>
          <a
            class="thelogin"
            href="/signup"
          >
            SIGN UP
          </a>
        </div>
      </header>
    </div>
  </body>,
  "container": <div>
    <header
      class="header"
    >
      <h1
        class="header-logo"
      >
        <a
          href="/"
        >
          <span
            class="q"
          >
            Q
          </span>
          <span
            class="uestioner"
          >
            UESTIONER
          </span>
           
        </a>
         
      </h1>
      <div
        class="profile"
      >
        <a
          class="thelogin"
          href="/login"
        >
          LOG IN
        </a>
        <a
          class="thelogin"
          href="/signup"
        >
          SIGN UP
        </a>
      </div>
    </header>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Should render components <HeaderLogin /> component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header
        class="header"
      >
        <h1
          class="header-logo"
        >
          <a
            href="../index.html"
          >
            <span
              class="q"
            >
              Q
            </span>
            <span
              class="uestioner"
            >
              UESTIONER
            </span>
          </a>
           
        </h1>
        <div
          class="profile"
        >
          <a
            class="thelogin"
            href="../admin/meetups.html"
            id="adminStatus"
          >
            ADMIN
             
          </a>
          <img
            alt=""
            id="dropdown"
            src="https://cdn.pixabay.com/photo/2016/08/08/09/17/avatar-1577909_960_720.png"
          />
          <br />
           
          <span
            class="time"
            id="currentTime"
          />
        </div>
      </header>
    </div>
  </body>,
  "container": <div>
    <header
      class="header"
    >
      <h1
        class="header-logo"
      >
        <a
          href="../index.html"
        >
          <span
            class="q"
          >
            Q
          </span>
          <span
            class="uestioner"
          >
            UESTIONER
          </span>
        </a>
         
      </h1>
      <div
        class="profile"
      >
        <a
          class="thelogin"
          href="../admin/meetups.html"
          id="adminStatus"
        >
          ADMIN
           
        </a>
        <img
          alt=""
          id="dropdown"
          src="https://cdn.pixabay.com/photo/2016/08/08/09/17/avatar-1577909_960_720.png"
        />
        <br />
         
        <span
          class="time"
          id="currentTime"
        />
      </div>
    </header>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Should render the component <Aside /> component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <aside
        class="aside"
      >
        <div
          class="aside-header"
        >
          Upcoming Meetups
        </div>
        <div
          class="aside-body"
        >
          <div
            class="aside-content upcoming-meetup"
          >
            <p>
               
              Today: 
              <span>
                 0
              </span>
            </p>
            <p>
               
              This Week:
              <span>
                 23
              </span>
            </p>
            <p>
               
              This Month:
              <span>
                 123
              </span>
            </p>
          </div>
          <div
            class="aside-header"
          >
             Your Stats
          </div>
          <div
            class="aside-body"
          >
            <div
              class="aside-content"
            >
              <p
                class="stats-list"
              >
                Question Posted: 
                <span>
                  10
                </span>
                 
              </p>
              <p
                class="stats-list"
              >
                 
                Question Commented On:
                <span>
                   23
                </span>
                 
              </p>
              <p
                class="stats-list"
              >
                 
                Total Upvote:P 
                <span>
                  123 
                </span>
                 
              </p>
              <p
                class="stats-list"
              >
                Total Downvote: 
                <span>
                  34 
                </span>
                 
              </p>
            </div>
          </div>
        </div>
        <div
          class="aside-header"
        >
          ADVERTS
        </div>
        <div
          class="aside-body"
        >
          <img
            alt="advert"
            src="https://i.imgur.com/XaKeYJE.png"
          />
        </div>
      </aside>
    </div>
  </body>,
  "container": <div>
    <aside
      class="aside"
    >
      <div
        class="aside-header"
      >
        Upcoming Meetups
      </div>
      <div
        class="aside-body"
      >
        <div
          class="aside-content upcoming-meetup"
        >
          <p>
             
            Today: 
            <span>
               0
            </span>
          </p>
          <p>
             
            This Week:
            <span>
               23
            </span>
          </p>
          <p>
             
            This Month:
            <span>
               123
            </span>
          </p>
        </div>
        <div
          class="aside-header"
        >
           Your Stats
        </div>
        <div
          class="aside-body"
        >
          <div
            class="aside-content"
          >
            <p
              class="stats-list"
            >
              Question Posted: 
              <span>
                10
              </span>
               
            </p>
            <p
              class="stats-list"
            >
               
              Question Commented On:
              <span>
                 23
              </span>
               
            </p>
            <p
              class="stats-list"
            >
               
              Total Upvote:P 
              <span>
                123 
              </span>
               
            </p>
            <p
              class="stats-list"
            >
              Total Downvote: 
              <span>
                34 
              </span>
               
            </p>
          </div>
        </div>
      </div>
      <div
        class="aside-header"
      >
        ADVERTS
      </div>
      <div
        class="aside-body"
      >
        <img
          alt="advert"
          src="https://i.imgur.com/XaKeYJE.png"
        />
      </div>
    </aside>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Should render the component <Footer /> component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <footer
        class="footer"
      >
        copyright @ Adebajo Oluwaseyi | Andela 2018
      </footer>
    </div>
  </body>,
  "container": <div>
    <footer
      class="footer"
    >
      copyright @ Adebajo Oluwaseyi | Andela 2018
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Should render the component <Header /> component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header
        class="header"
      >
        <h1
          class="header-logo"
        >
          <a
            href="/"
          >
            <span
              class="q"
            >
              Q
            </span>
            <span
              class="uestioner"
            >
              UESTIONER
            </span>
             
          </a>
           
        </h1>
        <div
          class="profile"
        >
          <a
            class="thelogin"
            href="/login"
          >
            LOG IN
          </a>
          <a
            class="thelogin"
            href="/signup"
          >
            SIGN UP
          </a>
        </div>
      </header>
    </div>
  </body>,
  "container": <div>
    <header
      class="header"
    >
      <h1
        class="header-logo"
      >
        <a
          href="/"
        >
          <span
            class="q"
          >
            Q
          </span>
          <span
            class="uestioner"
          >
            UESTIONER
          </span>
           
        </a>
         
      </h1>
      <div
        class="profile"
      >
        <a
          class="thelogin"
          href="/login"
        >
          LOG IN
        </a>
        <a
          class="thelogin"
          href="/signup"
        >
          SIGN UP
        </a>
      </div>
    </header>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Should render the component <HeaderLogin /> component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header
        class="header"
      >
        <h1
          class="header-logo"
        >
          <a
            href="../index.html"
          >
            <span
              class="q"
            >
              Q
            </span>
            <span
              class="uestioner"
            >
              UESTIONER
            </span>
          </a>
           
        </h1>
        <div
          class="profile"
        >
          <a
            class="thelogin"
            href="../admin/meetups.html"
            id="adminStatus"
          >
            ADMIN
             
          </a>
          <img
            alt=""
            id="dropdown"
            src="https://cdn.pixabay.com/photo/2016/08/08/09/17/avatar-1577909_960_720.png"
          />
          <br />
           
          <span
            class="time"
            id="currentTime"
          />
        </div>
      </header>
    </div>
  </body>,
  "container": <div>
    <header
      class="header"
    >
      <h1
        class="header-logo"
      >
        <a
          href="../index.html"
        >
          <span
            class="q"
          >
            Q
          </span>
          <span
            class="uestioner"
          >
            UESTIONER
          </span>
        </a>
         
      </h1>
      <div
        class="profile"
      >
        <a
          class="thelogin"
          href="../admin/meetups.html"
          id="adminStatus"
        >
          ADMIN
           
        </a>
        <img
          alt=""
          id="dropdown"
          src="https://cdn.pixabay.com/photo/2016/08/08/09/17/avatar-1577909_960_720.png"
        />
        <br />
         
        <span
          class="time"
          id="currentTime"
        />
      </div>
    </header>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
